ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/plugin/ManifestEditor.java

```diff
@@ -356,9 +350,19 @@ public void editorContextAdded(InputContext context) {
 		}
 	}
 
+	@Override
+	public void doSave(IProgressMonitor monitor) {
+		// TODO Auto-generated method stub
+		super.doSave(monitor);
+	}
+
```

Comment:
Is this still necessary? It looks obsolete.

Reply:
Removed